### ISSUE TITLE
version bounds for extra are too strict (broken on NixOS, breaks matterhorn)

### ISSUE THREAD

**Issue by Kiwi** [kapralVV/Unique#5]
I'm the [maintainer for matterhorn (and by proxy Unique)](https://github.com/NixOS/nixpkgs/blob/9bd0df3ad16eb4a7ccafedd7300823bc5aef95b4/pkgs/development/haskell-modules/configuration-hackage2nix.yaml#L2646-L2651) on NixOS, which uses stackage, and the version of extra on stackage is higher than Unique allows in the constraints, so Unique fails to build. Which makes matterhorn fail to build. Which makes me sad. and have to make lots of PRs (this is not the first PR I've had for this) :(

[PR (link directly to a comment with explanation)](https://github.com/NixOS/nixpkgs/pull/92618#discussion_r451303303)

tl;dr they suggest getting Unique/matterhorn on stackage as the best long term solution, alternatively if you update the constraints in the Unique cabal file on hackage to allow newer versions of extra that would help too and is probably easier.

I don't know if you are interested in putting it on stackage or how that even works or if someone else can do it for you or not. Is that something you would consider?

Thanks

**Comment by kapralVV** [comment]
@Kiwi  
I've added you as a collaborator of this repo. The invitation is sent . 
Please feel free to add/fix everything you need. 
Thanks for you work and supporting NixOS packages
